packages/blade/src/components/Tabs/__tests__/Tabs.web.test.tsx

```diff
@@ -0,0 +1,446 @@
+    const { baseElement } = renderWithTheme(
+      <Tabs defaultValue="refunds" size="large">
+        <TabList>
+          <TabItem value="payments">Payments</TabItem>
+          <TabItem value="refunds">Refunds</TabItem>
+        </TabList>
+
+        <TabPanel value="payments">
+          <Text>Payments</Text>
+        </TabPanel>
+        <TabPanel value="refunds">
+          <Text>Refunds</Text>
+        </TabPanel>
+      </Tabs>,
+    );
+
+    expect(baseElement).toMatchSnapshot();
+  });
+
+  it('should switch tabs with mouse', async () => {
+    const user = userEvent.setup();
+    const { getByRole } = renderWithTheme(
+      <Tabs>
+        <TabList>
+          <TabItem value="payments">Payments</TabItem>
+          <TabItem value="refunds">Refunds</TabItem>
+          <TabItem value="disputes">Disputes</TabItem>
+        </TabList>
+
+        <TabPanel value="payments">
+          <Text>Payments</Text>
+        </TabPanel>
+        <TabPanel value="refunds">
+          <Text>Refunds</Text>
+        </TabPanel>
+        <TabPanel value="disputes">
+          <Text>Disputes</Text>
+        </TabPanel>
+      </Tabs>,
+    );
+
+    expect(getByRole('tab', { name: 'Payments' })).toHaveAttribute('aria-selected', 'true');
+
+    // TODO: change this after isLazy implementation
+    expect(queryTabPanelByText('Payments')).toBeVisible();
+    expect(queryTabPanelByText('Refunds')).not.toBeVisible();
+    expect(queryTabPanelByText('Disputes')).not.toBeVisible();
+
+    await user.click(getByRole('tab', { name: 'Refunds' }));
+    expect(getByRole('tab', { name: 'Refunds' })).toHaveAttribute('aria-selected', 'true');
+    expect(queryTabPanelByText('Payments')).not.toBeVisible();
+    expect(queryTabPanelByText('Refunds')).toBeVisible();
+    expect(queryTabPanelByText('Disputes')).not.toBeVisible();
+
+    await user.click(getByRole('tab', { name: 'Disputes' }));
+    expect(getByRole('tab', { name: 'Disputes' })).toHaveAttribute('aria-selected', 'true');
+    expect(queryTabPanelByText('Payments')).not.toBeVisible();
+    expect(queryTabPanelByText('Refunds')).not.toBeVisible();
+    expect(queryTabPanelByText('Disputes')).toBeVisible();
+  });
+
+  it('should switch tabs with keyboard', async () => {
+    const user = userEvent.setup();
+    const { getByRole } = renderWithTheme(
+      <Tabs>
+        <TabList>
+          <TabItem value="payments">Payments</TabItem>
+          <TabItem value="refunds">Refunds</TabItem>
+          <TabItem value="disputes">Disputes</TabItem>
+        </TabList>
+
+        <TabPanel value="payments">
+          <Text>Payments</Text>
+        </TabPanel>
+        <TabPanel value="refunds">
+          <Text>Refunds</Text>
+        </TabPanel>
+        <TabPanel value="disputes">
+          <Text>Disputes</Text>
+        </TabPanel>
+      </Tabs>,
+    );
+
+    expect(getByRole('tab', { name: 'Payments' })).toHaveAttribute('aria-selected', 'true');
+
+    // TODO: change this after isLazy implementation
```

Comment:
isLazy is implemented now right? anything needs to change here?

Reply:
Yeah this is the modified tests only, forgot to remove the comment. 

I've also added isLazy prop test separately.